starter-code/index.js

```diff
@@ -1,11 +1,45 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
-  };
+    constructor() {
+      this.items=[];
+      this.length=0
+    }
+    add(item) {
+      this.items.push(item);
+     this.items.sort((a,b)=>(a-b))
+      this.length +=1;
+
+    }
+    get(pos) {
+        return this.items[pos];
+
+    }
+    max() {
+      if (this.length===0){
+        throw new Error("EmptySortedList")
+      } else return this.items [this.length-1]
+
```

Comment:
Ici, quand on demande de return le 'max element in the list', on parle du chiffre le plus élevé.
Or ici, ton code retourne le dernier élément du tableau, peu importe qu'il soit le plus grand ou pas.
On aurait plutôt dû avoir quelque chose du genre :
```javascript
else return this.items.reduce((bigger, current) => {
  if (current > bigger) {return current}
  else return bigger
}, 0)
```

Reply:
Ah merci Lucas, en effet j'avais compris "maximum" comme dernier élément ... ( n'importequoi en plus...je vais prendre des cours de français aussi je crois haha ) .